bin/webpack-dev-server.js

```diff
@@ -310,8 +310,8 @@ function processOptions(wpOpt) {
 	if(argv["compress"])
 		options.compress = true;
 
-	if(argv["open"])
-		options.open = true;
+	if(argv["open"] !== undefined)
```

Comment:
It's not entirely obvious but there are three potential values that could come from argv["open"]. It could be undefined (meaning the user did not specify the --open option), it could be an empty string (meaning the user added the --open option without a file path), or it could be a string containing a path (meaning the user added the --open option with a file path). The intent here is to only capture the value when it's a blank string or a string containing characters. That's not entirely obvious so I've changed it to the truthy check or a check for a blank string. As I'm thinking about this I could use a typeof string check instead. I can switch it over if you'd prefer.